framework_tvb/tvb/adapters/simulator/model_forms.py

```diff
@@ -517,6 +529,116 @@ def __init__(self, prefix=''):
+
+class CoombesByrneModelForm(FormWithRanges):
+
+    def __init__(self, prefix=''):
+        super(CoombesByrneModelForm, self).__init__(prefix)
+        self.Delta = ArrayField(ModelsEnum.COOMBES_BYRNE.get_class().Delta, self)
+        self.alpha = ArrayField(ModelsEnum.COOMBES_BYRNE.get_class().alpha, self)
+        self.v_syn = ArrayField(ModelsEnum.COOMBES_BYRNE.get_class().v_syn, self)
+        self.k = ArrayField(ModelsEnum.COOMBES_BYRNE.get_class().k, self)
+        self.eta = ArrayField(ModelsEnum.COOMBES_BYRNE.get_class().eta, self)
+        self.variables_of_interest = MultiSelectField(ModelsEnum.COOMBES_BYRNE.get_class().variables_of_interest, self)
+
+    @staticmethod
+    def get_params_configurable_in_phase_plane():
+        return ['Delta', 'alpha', 'v_syn', 'k', 'eta', 'variables_of_interest']
+
+class CoombesByrne2DModelForm(FormWithRanges):
+
+    def __init__(self, prefix=''):
+        super(CoombesByrne2DModelForm, self).__init__(prefix)
+        self.Delta = ArrayField(ModelsEnum.COOMBES_BYRNE_2D.get_class().Delta, self)
+        self.v_syn = ArrayField(ModelsEnum.COOMBES_BYRNE_2D.get_class().v_syn, self)
+        self.k = ArrayField(ModelsEnum.COOMBES_BYRNE_2D.get_class().k, self)
+        self.eta = ArrayField(ModelsEnum.COOMBES_BYRNE_2D.get_class().eta, self)
+        self.variables_of_interest = MultiSelectField(ModelsEnum.COOMBES_BYRNE_2D.get_class().variables_of_interest, self)
+
+    @staticmethod
+    def get_params_configurable_in_phase_plane():
+        return ['Delta', 'v_syn', 'k', 'eta', 'variables_of_interest']
+
+class GastSchmidtKnosche_SDModelForm(FormWithRanges):
+
+    def __init__(self, prefix=''):
+        super(GastSchmidtKnosche_SDModelForm, self).__init__(prefix)
+        self.tau = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().tau, self)
+        self.tau_A = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().tau_A, self)
+        self.alpha = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().alpha, self)
+        self.I = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().I, self)
+        self.Delta = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().Delta, self)
+        self.J = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().J, self)
+        self.eta = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().eta, self)
+        self.cr = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().cr, self)
+        self.cv = ArrayField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().cv, self)
+        self.variables_of_interest = MultiSelectField(ModelsEnum.GAST_SCHMIDT_KNOSCHE_SD.get_class().variables_of_interest, self)
+
+    @staticmethod
+    def get_params_configurable_in_phase_plane():
+        return ['tau', 'tau_A', 'alpha', 'I', 'Delta', 'J', 'eta', 'cr', 'cv', 'variables_of_interest']
+
+class GastSchmidtKnosche_SFModelForm(FormWithRanges):
```

Comment:
I thought it is not a good practice to have underscores in class names, but I saw it's the same way in the library.